README.md

```diff
@@ -111,7 +111,9 @@ Build Go CD agent with AWS profile configured
 
   roles:
     - name: sansible.gocd_agent
-      sansible_gocd_agent_aws_gocd_server_lookup_filter: "Name=tag:environment,Values=prd Name=tag:role,Values=gocd_server"
+      sansible_gocd_agent_aws_gocd_server_lookup_filters:
+        "tag:environment": prd
+        "tag:role": gocd_server
```

Comment:
I would add another example, such as

```yaml
instance-type: t2.small
```
to show that it can be something else than just a tag.

Reply:
:+1: